src/portable_python/external/tkinter.py

```diff
@@ -107,7 +108,7 @@ class TkInter(ModuleBuilder):
     """
 
     m_debian = "-tk-dev"
-    m_telltale = ["{include}/tk", "{include}/tk.h"]
+    m_telltale: ClassVar[list] = ["{include}/tk", "{include}/tk.h"]
```

Comment:
I'm not sure I buy `ClassVar[list]` as being a full type.  Shouldn't it specify the type the list contains, like `ClassVar[List[str]]` (and with capital L until 3.7 with a future import, and new enough mypy).  See https://peps.python.org/pep-0585/#implementation for where I think this is defined.

Reply:
I think I'll revisit this and get rid of the `ClassVar`, I think simply using tuples is better (don't remember why I went with `list`)